### PR TITLE
[tests-only] test: add e2e test for directly opening project space through url

### PR DESCRIPTION
**Pull request by SwikritiT**
## Related Issue

- Part of: https://github.com/owncloud/web/issues/9897

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/e2e/cucumber/features/smoke/urlJourneys.feature [hunk 1]
```diff
@@ -19,21 +19,38 @@ Feature: web can be navigated through urls
...
+      | name              | content                   |
+      | spaceTextfile.txt | This is test file. Cheers |
+    When "Alice" logs in
+    And "Alice" opens space "Development" through the URL
+    And "Alice" opens the following file in texteditor
+      | resource          |
+      | spaceTextfile.txt |
+    Then "Alice" is in a text-editor
+    And "Alice" closes the file viewer
+    Then "Alice" logs out
```
Comment (ScharfViktor) [inline tests/e2e/cucumber/features/smoke/urlJourneys.feature:56]: I would create separate scenario here? or move please creating space to top of the test and delete logs out step

Comment (SwikritiT) [reply tests/e2e/cucumber/features/smoke/urlJourneys.feature]: I've combined them into a single one

File: tests/e2e/cucumber/steps/ui/navigateByUrl.ts [hunk 2]
```diff
@@ -3,26 +3,37 @@ import { World } from '../../environment'
...
     const urlNavObject = new objects.urlNavigation.URLNavigation({ page })
-    await urlNavObject.navigateToDetailsPanelOfResource({ resource, detailsPanel, user })
+    await urlNavObject.navigateToDetailsPanelOfResource({ resource, detailsPanel, user, space })
   }
 )
 
 When(
-  '{string} opens the file/folder {string} through the URL',
-  async function (this: World, stepUser: string, resource: string): Promise<void> {
+  /^"([^"]*)" opens the (?:resource|file|folder)? "([^"]*)" of space "([^"]*)" through the URL$/,
```
Comment (saw-jan) [inline tests/e2e/cucumber/steps/ui/navigateByUrl.ts:22]:
```suggestion
  /^"([^"]*)" opens the (?:resource|file|folder) "([^"]*)" of space "([^"]*)" through the URL$/,
```

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Review by ScharfViktor — APPROVED** [review]

**Pull request merged**